packages/app/.env.client.development

```diff
@@ -1,6 +0,0 @@
-# Unset the following EARLY_PREVIEW var to see the default UI as it is deployed to the cluster.
-# This variable is unset in the cluster so users only receive new features once they're ready for consumption.
-# EARLY_PREVIEW: "true"
-
-AUTH0_CLIENT_ID: vs6bgTunbVK4dvdLRj02DptWjOmAVWVM
-AUTH0_DOMAIN: opstrace-dev.us.auth0.com
```

Comment:
Existence of this file was misleading: isn't consumed in the client

Reply:
We do still want to keep the EARLY_PREVIEW flag because that's allows one to work on features that aren't yet released. The others are indeed not used.